### ISSUE TITLE
Define and create list of 'reference pages' required for accessibility audit

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1160]
## What

Define a list of 'reference pages' that, between all of the examples, provide coverage for every style, component and component variant that we would want to be audited.

(For example, buttons _and_ 'link' buttons; radios with and without hints)

## Why

We are planning to get Frontend audited to ensure it is compliant with WCAG 2.1 (we had an audit before the Design System launched in early 2018, but against WCAG 2.0)

We believe that the audit is going to identify more issues (and avoid flagging 'false positives') if we audit the components in a realistic context. For example, the error summary component only really 'makes sense' if it is used in the context of a form.

## Done when…

- [x] We have a list of styles, components and component variants that we want to be audited
- [x] We have a definitive list of screens that need to be built in order to use each of those styles, components and component variants in a realistic context
- [x] We have tested all pages with axe (accessibility automation software) and html validator
- [x] We have added 'Error: ' to the title of pages that have errored.
- [x] We have explained the user journeys that should be tested (for example errors)
- [x] We have double checked all pages with someone in the team following the user journeys
- [x] We have actioned changes from feedback from member of team who did the testing

**Comment by nickcolley** [comment]
This is done so closing
